soccer/MainWindow.cpp

```diff
@@ -362,6 +362,7 @@ void MainWindow::updateViews() {
         _procFPS->setText(
             QString("Proc: %1 fps").arg(_processor->framerate(), 0, 'f', 1));
 
+        // NOTE: why are we using magic numbers instead of defined constants?
```

Comment:
Probably should be a TODO instead of a NOTE, unless you just want to fix it now.